Spring-Boot/Spring-Boot-Academy/src/test/java/tw/waterball/judgegirl/springboot/academy/controllers/HomeworkControllerTest.java

```diff
@@ -99,6 +102,27 @@ public void GivenOneHomeworkCreated_WhenGetHomeworkById_ShouldRespondHomework()
         assertEquals(homework, actualHomework);
     }
 
+    @Test
+    public void GivenOneHomeworkCreated_WhenDeleteHomeworkById_ShouldSucceed() throws Exception {
+        var homework = createHomeworkAndGet(HOMEWORK_NAME);
+
+        deleteHomework(homework.id).andExpect(status().isOk());
+
+        assertFalse(homeworkRepository.findHomeworkById(homework.id).isPresent());
+    }
+
+    @Test
+    public void GivenManyHomeworkCreated_WhenGetAllHomework_ShouldReturnAllHomework() throws Exception {
```

Comment:
`Given3HomeworkCreated_WhenGetAllHomework_ShouldRespondHomeworkListOfSize3`

**Specification by example**, prefer the actual number **3**